### ISSUE TITLE
Can't play video in Android

### ISSUE THREAD

**Issue by windystrife** [google-ar/WebARonARCore#71]
i write some demo to play video .mp4 (no AR) , three.js only 
but it can't play video in webARonARCore (it's black screen)
can play well on PC.
here it is: 
https://windystrife.github.io/three.ar.js/examples/VideoTest.html

(it can work with livestream m3u8, but mp4 can't)

**Comment by robertjuh** [comment]
if it does work on normal chrome browsers it could be an issue with the current version of Chromium that webarONarcore uses.

**Comment by windystrife** [comment]
yeah it work with normal chrome browers, but this version for ARCore not work

**Comment by robertjuh** [comment]
Yes i know. 
What i do know is that they are working on trying to bring three.ar.js to a newer version of Chromium. There might be a chance that this would change the matter.

**Comment by robertjuh** [comment]
or maybe the DOM is different? have you tried the remote debugger function from chrome? 
It is possible to debug what's happening on your phone by connecting to a remote device

**Comment by Danpollak** [comment]
Experienced the same issue when trying to render video through aframe-xr on this browser.
Got "Uncaught (in promise) DOMException: Failed to load because no supported source was found."
There was a user interaction to play the video, and I couldn't get it to play also through debugging and forced play() through the console.
The video did work on iOS at AR environment.
Tested on Pixel 2.

**Comment by judax** [comment]
This build of chromium does not have support for all the video formats. We will try to resolve this in the new completely rebuilt version we are working on. Sorry for the inconveniences.
